language-plutus-core/src/Language/PlutusCore/Constant/Typed.hs

```diff
@@ -161,24 +106,22 @@ data TypeScheme size a r where
            -- a type constructor to the variable, like in
            --
            -- > reverse : all a. list a -> list a
-        -> (forall qt. qt ~ OpaqueTerm text uniq => TypedBuiltin size qt -> TypeScheme size a r)
-        -> TypeScheme size a r
-    TypeSchemeAllSize :: (size -> TypeScheme size a r) -> TypeScheme size a r
+        -> (forall qt. qt ~ OpaqueTerm text uniq => TypedBuiltin qt -> TypeScheme a r)
```

Comment:
While you're here, could you please rename `qt` to `ot` (it was `QuotedTerm` originally)?